src/AdminConsole/Pages/App/Settings/Settings.cshtml

```diff
@@ -89,7 +89,7 @@
     <panel header="Attestation">
         <p>When attestation is enabled, you can choose to limit the authenticators used with your application.</p>
         
-        <div>
+        <div class="flex">

```

Comment:
I think we'd rather change the implementation of `.btn-primary` to use `display:inline-block`